lib/tasks/search_event_loader.rake

```diff
@@ -14,16 +14,28 @@ namespace :search_events do
   # @example
   # bin/rails search_events:csv_loader['local_path_to_file.csv', 'some-source-to-use-for-all-loaded-records']
   #
+  # @example
```

Comment:
@matt-bernhardt I added a commit to consolidate to a single task that takes either a local file or remote URL to match how `openalex_loader` works. I'll open a ticket to take a look at `suggested_resources:reload` as well later as it notes it can only handle remote and I'd like all of these to behave similarly.

Let me know if you have any concerns with this refactor.

Reply:
Yeah, I'm definitely comfortable with this in the short term, and over the long term I agree with making all these loaders resemble each other. Thanks!